app/graphql/generated.gql

```diff
@@ -373,6 +373,17 @@ mutation userContactUpdateAlias($input: UserContactUpdateAliasInput!) {
   }
 }
 
+mutation userDeviceAccountCreate {
+  userDeviceAccountCreate {
+    success
```

Comment:
Will success be true if the account is already created? I think it would be helpful for that to be the case because when a user logs out and then tries to log back into a device account, this mutation will be hit again.